index.js

```diff
@@ -218,6 +220,7 @@ Lexer.prototype = {
       var tok = this.tok('comment', captures[2]);
       tok.buffer = '-' != captures[1];
       this.pipeless = true;
+      this.interpolationAllowed = tok.buffer;
```

Comment:
I think that maybe we should push people to write:

```html
<!-- whatever they want here -->
```

when they are actually using comments for some kind of inner templating language (like knockout) and reserve

```js
// whatever here
```

for true comments.  This matches our stance on conditional comments and can be messaged appropriately in the docs.